organizations/creativecommons.md

```diff
@@ -0,0 +1,35 @@
+# Creative Commons Overview
+
+**Website:**  
+[Creative Commons About](https://creativecommons.org/about/)
+
+**Resources:**  
+- [Creative Commons Resources](https://creativecommons.org/share-your-work/)  
+- [Open Education page](https://creativecommons.org/about/education/)
+
+
+## Overview
+Creative Commons (CC) is a global non-profit organization that provides free, legally sound tools to enable sharing, collaboration, and reuse of creative works. Through its widely used open licenses, Creative Commons facilitates open access to content while ensuring that authors can maintain control over how their work is used. Its mission is to promote knowledge-sharing and creativity across academia, art, science, education, and government sectors.
+
+## Contact
+For general inquiries, please reach out to [info@creativecommons.org](mailto:info@creativecommons.org) 
+For press/media inquiries, contact: [communications@creativecommons.org](mailto:communications@creativecommons.org).
+
+**Personnel:** CEO Anna Tumadóttir supported by staff, Board Members, Advisory Council, and Audit Committee, along with a global community of affiliates, volunteers, and legal experts.
```

Comment:
I don't think volunteers should be included here. 

What do we think about this section? Could it be improved?

I'm not sure bolding a header makes sense - shouldn't it be `###`?

Reply:
> I don't think volunteers should be included here.
> 
> What do we think about this section? Could it be improved?
> 
> I'm not sure bolding a header makes sense - shouldn't it be `###`?

Removed